grakn-graql/src/main/java/ai/grakn/graql/internal/analytics/DegreeDistributionMapReduce.java

```diff
@@ -47,14 +47,16 @@
     public DegreeDistributionMapReduce() {
     }
 
-    public DegreeDistributionMapReduce(Set<TypeId> selectedTypeIds) {
+    public DegreeDistributionMapReduce(Set<TypeId> selectedTypeIds, String degreeLabel) {
```

Comment:
For the sake of clarity `degreeLabel` should be a `TypeLabel`. Makes the method more readable and easier to work with form the outsider's perspective.